docs/daisy-migrating-existing-workflows.md

```diff
@@ -0,0 +1,115 @@
+# Migrating Image Creation Workflows to Daisy
```

Comment:
I'm thinking we should call this something other than "migrating" - only because it can be confused with VM and image migration (which is a thing we are doing with Daisy). Maybe we frame this as just using Daisy for image creation workflows. I get the point about assuming there are previous workflows for other tools but there are so many variations that we may as well just be general here.

A useful future guide however would be how to convert a packer workflow into a daisy workflow as that is one of the more common tools used.

Reply:
True, this page may be too specialized to be generally useful. I did indeed write it with a certain audience in mind.

I'll change it to be more generally about using Daisy to create images.

Reply:
Okay, done. It actually took fewer changes than I thought -- I barely had to change anything. The tone of the doc is now a little more open to the possibility that you're currently building your images by hand, rather than assuming you already had an automated solution.

The title of the page (and thus the name of the file) has been changed to "Automating Image Creation with Daisy".